tests/e2e/cluster_upgrade_test.go

```diff
@@ -86,45 +90,53 @@ var _ = Describe("Upgrade", func() {
 				Expect(err).ToNot(HaveOccurred())
 
 				By("Wait the upgrade finished")
-				err = openshift.WaitClassicClusterUpgradeFinished(ci.RHCSConnection, clusterID)
+				err = openshift.WaitClassicClusterUpgradeFinished(cms.RHCSConnection, clusterID)
 				Expect(err).ToNot(HaveOccurred(), "Cluster upgrade %s failed with the error %v", clusterID, err)
 
 				By("Wait for 10 minutes to be sure the version is synced in clusterdeployment")
 				time.Sleep(10 * time.Minute)
 
 				By("Check the cluster status and OCP version")
-				clusterResp, err = cms.RetrieveClusterDetail(ci.RHCSConnection, clusterID)
+				clusterResp, err = cms.RetrieveClusterDetail(cms.RHCSConnection, clusterID)
 				Expect(err).ToNot(HaveOccurred())
 				Expect(string(clusterResp.Body().State())).To(Equal(constants.Ready))
 				Expect(string(clusterResp.Body().Version().RawID())).To(Equal(targetV))
 
-				if constants.GetEnvWithDefault(constants.WaitOperators, "false") == "true" && !profile.Private {
+				if constants.GetEnvWithDefault(constants.WaitOperators, "false") == "true" && !profile.IsPrivate() {
 					// WaitClusterOperatorsToReadyStatus will wait for cluster operators ready
 					timeout := 60
-					err = openshift.WaitForOperatorsToBeReady(ci.RHCSConnection, clusterID, timeout)
+					err = openshift.WaitForOperatorsToBeReady(cms.RHCSConnection, clusterID, timeout)
 					Expect(err).ToNot(HaveOccurred())
 				}
 			})
 
 		It("on Y-stream - [id:63152]", ci.Upgrade,
 			func() {
-
-				if profile.VersionPattern != "y-1" {
+				profile := profile
```

Comment:
why this line is needed?

Reply:
I think a replace all did that :D nice catch !